src/scenes/AcceleratorRouter/ParticipantProjects/ListView.tsx

```diff
@@ -192,6 +192,7 @@ export default function ListView(): JSX.Element {
         rightComponent={actionMenus}
         rows={projects}
         title={strings.PROJECTS}
+        stickyFilters={true}
```

Comment:
Just a nit, but we can omit the `={true}`:

```suggestion
        stickyFilters
```